### ISSUE TITLE
NonRandomRespondent does not select multiple from "checkbox" questions

### ISSUE THREAD

**Issue by jjfiv** [SurveyMan/SurveyMan#501]
RandomRespondent explicitly chooses a "bitset" at random from the space of choices. Not sure the right model here. @etosch

**Comment by etosch** [comment]
Yeah, NonRandomRespondent should have an implementation similar to RandomRespondent.

**Comment by jjfiv** [comment]
Might make sense to have each option be an independent trial rather than keeping a distribution over 2^{numOptions} bitsets for a preference. I'm not sure of the modeling implications, though.

**Comment by etosch** [comment]
Well, then you will get subsets of the expected size m/2, and it would very rarely be the case that you would only choose one or choose all.
